### PR TITLE
[VPU] Support for originalLayersNames attribute in exec graph

### PR DESCRIPTION
**Pull request by AndrewBakalinIntel**
Ticket - #-32838

### CODE REVIEW

File: inference-engine/src/vpu/graph_transformer/src/backend/get_meta_data.cpp [hunk 1]
```diff
@@ -84,7 +84,11 @@ void BackEnd::getMetaData(
             stageMeta.layerName = "<Extra>";
             stageMeta.layerType = "<Extra>";
         } else {
-            stageMeta.layerName = stage->origLayer()->name;
+            if (stage->origLayer()->params.count("originalLayersNames")) {
+                stageMeta.layerName = stage->origLayer()->GetParamAsString("originalLayersNames");
+            } else {
+                stageMeta.layerName = stage->origLayer()->name;
+            }
```
Comment (gladilov-gleb) [inline inference-engine/src/vpu/graph_transformer/src/backend/get_meta_data.cpp:91]:
```suggestion
            const auto& original = stage->origLayer();
            stageMeta.layerName = original->GetParamAsString("originalLayersNames", original->name);
```

**Comment by gladilov-gleb** [comment]
What is the root cause for CI failure?

**Comment by AndrewBakalinIntel** [comment]
> What is the root cause for CI failure?

@gladilov-gleb Looks like some sort of build failure. I will take a look at this closely and assign to you once everything is ready.

**Comment by AndrewBakalinIntel** [comment]
@GlebKazantaev @adudchen @gladilov-gleb 
After last suggestion fix, it turned out that I can't call `GetParamAsString("originalLayersNames", originalLayer->name)`, as both parameters are `const char*`. The one way was to convert the second parameter to `const char*`, but I decided to replace it with equivalent, just because we actually don't need any convertion here while getting these parameter, so we can take it as is.
Please take another look and doubt if you have any concerns.

**Comment by AndrewBakalinIntel** [comment]
@gladilov-gleb could you please take a look and merge if everything is ok? Internal validation failed due to the same issue as was reported on master branch.

**Pull request merged**